### PR TITLE
Updated Privacy policy 🔏

### PR DESCRIPTION
**Pull request by heartlog**
By using #3268 and README.md as reference.

### CODE REVIEW

**Comment by heartlog** [comment]
@Bnyro check the privacy policy update in Patch-1 branch. Updated as per FAQ answers from #3280 comment.

**Comment by faisalcodes** [comment]
It is better to put the privacy policy in a separate md file. Keeping the root readme clean will be good for visitors.

**Comment by heartlog** [comment]
@faisalcodes hope it is what you expected.

File: README.md [hunk 1]
```diff
@@ -141,14 +141,10 @@ Both are great clients for watching YouTube videos. It depends on the individual
 Privacy Policy and Disclaimer
 </h2>
 
-The LibreTube project aims to provide a private, anonymous experience for using web-based media services. Therefore, the app does not collect any data without your consent.
 
-The LibreTube project and its contents are not affiliated with, funded, authorized, endorsed by, or in any way associated with YouTube, Google LLC, or any of its affiliates or subsidaries.<br>
-Any trademark, service mark, trade name, or other intellectual property rights used are owned by the respective owners.
+At LibreTube, we are committed to protecting the privacy of our users. [Privacy Policy](/PRIVACY_POLICY.md) outlines the types of information application collect, how we use it, and how it protect your personal information. [Read Privacy policy](/PRIVACY_POLICY.md)
```
Comment (Bnyro) [inline README.md:145]:
```suggestion
LibreTube aims to protect the privacy of its users. [Our Privacy Policy](/PRIVACY_POLICY.md) gives detailed information on which data the app stores in order to work, how it is being used, and how the project protects your personal information. It is recommended to read the privacy policy of LibreTube as well as the privacy policy of the instance you have chosen inside the app.
```

File: PRIVACY_POLICY.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<h2 align="left">
+<sub>
+<img  src="fastlane/metadata/android/en-US/images/readme/privacy.svg"
+      height="30"
+      width="30">
+</sub>
+The LibreTube project - Privacy Policy
```
Comment (Bnyro) [inline PRIVACY_POLICY.md:7]:
```suggestion
LibreTube - Privacy Policy
```

File: PRIVACY_POLICY.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+</h2>
+
+``At LibreTube, we are committed to protecting the privacy of our users. This Privacy Policy outlines the types of information application collect, how we use it, and how it protect your personal information.``
```
Comment (Bnyro) [inline PRIVACY_POLICY.md:10]:
```suggestion
``LibreTube aims to protect the privacy of its users. [Our Privacy Policy](/PRIVACY_POLICY.md) gives detailed information on which data the app stores in order to work, how it is being used, and how the project protects your personal information. It is recommended to read the privacy policy of LibreTube as well as the privacy policy of the instance you have chosen inside the app.``
```

File: PRIVACY_POLICY.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+### Information Collection
+LibreTube only collects necessary data such as watch history, settings, and bookmarks. Users can choose to turn off data collection features. The app does not collect any data about how it is used or profile its users, and app also avoids using libraries that could track usage data.
+
+### Information Use
+LibreTube keeps your data only on your device and does not share it with anyone else. Your data is protected by encryption to prevent unauthorized access.
+
+### Open Source Software
+LibreTube is an open source software that is built for learning and research purposes. We welcome and encourage collaboration and contributions from the community to help improve the app.
+
+### Source Instances
```
Comment (Bnyro) [inline PRIVACY_POLICY.md:21]:
```suggestion
### Instance list source
```

File: PRIVACY_POLICY.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+Piped accounts can be completely deleted with no history kept, providing increased security for users on the site.
+
+### Data Security
+LibreTube connects to Piped instances instead of directly to YouTube, providing an additional layer of privacy. However, trust is shifted to the instance owner. Piped instances do not log user-specific data, but there is no guarantee regarding the actions of the instance owner. Even if logging occurred, the data would be less than that collected by YouTube since LibreTube does not send telemetry data to the instance's server.
+
+### Disclaimer
+The LibreTube project is not affiliated, authorized, or endorsed by YouTube, Google LLC, or any of its affiliates or subsidiaries. Any intellectual property used is owned by the respective owners.
+
+### Changes to Privacy Policy
+By using the LibreTube app, you consent to the handling of your information as outlined in this Privacy Policy. If you have any questions or concerns about our Privacy Policy, you are free to ask us question.
```
Comment (Bnyro) [inline PRIVACY_POLICY.md:37]:
```suggestion
By using the LibreTube app, you consent to the handling of your information as outlined in this Privacy Policy. If you have any questions or concerns about our Privacy Policy, you are free to open an issue to ask about it or ask at the official Matrix room of the app.
```

File: PRIVACY_POLICY.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+The LibreTube project - Privacy Policy
+</h2>
+
+``At LibreTube, we are committed to protecting the privacy of our users. This Privacy Policy outlines the types of information application collect, how we use it, and how it protect your personal information.``
+
+### Information Collection
+LibreTube only collects necessary data such as watch history, settings, and bookmarks. Users can choose to turn off data collection features. The app does not collect any data about how it is used or profile its users, and app also avoids using libraries that could track usage data.
+
+### Information Use
+LibreTube keeps your data only on your device and does not share it with anyone else. Your data is protected by encryption to prevent unauthorized access.
```
Comment (Bnyro) [inline PRIVACY_POLICY.md:16]:
```suggestion
LibreTube doesn't share your data with 3rd parties. If you use the app without being logged in with an account, no relevant data will be shared with the Piped servers at all, it'll only be stored locally on the device. When being logged in with a Piped account, your subscriptions and playlists are saved to your Piped account.
```

**Comment by heartlog** [comment]
@Bnyro It's much better !
Get other maintainers to inspect and suggest the changes, and more importantly approve it.

**Comment by Bnyro** [comment]
Why did you remove the request for Kavin and Isira?

**Comment by heartlog** [comment]
> Why did you remove the request for Kavin and Isira?

Oh sorry 😐 my pointer clicked something, so that's it was..I apologise It's my mistake

**Review by Isira-Seneviratne — APPROVED** [review]

**Comment by Isira-Seneviratne** [comment]
Looks good to me.

**Review by FireMasterK — COMMENTED** [review]
I feel there's a lack of clarification on what is actually sent to Piped instances.

On a side note, maybe we add a privacy policy configuration that could be used by LibreTube/Piped? (it would be set up by instance owners)

**Comment by heartlog** [comment]
> I feel there's a lack of clarification on what is actually sent to Piped instances.
> 
> On a side note, maybe we add a privacy policy configuration that could be used by LibreTube/Piped? (it would be set up by instance owners)

It's mentioned there subscription, playlist & video. And according to @Bnyro, user is able to decide data can or can't shared to server. If you have any changes just commit it then we'll figure out what's missing.

**Comment by heartlog** [comment]
@ynm2ps2jfp87nkl All other changes are made successfully. But, last one just bothers me.

It's not implemented 'yet', I guess ?
Can can you explain in short what #3290 even mean and it's purpose.

**Comment by heartlog** [comment]
> I feel there's a lack of clarification on what is actually sent to Piped instances.

**It's done !**

> On a side note, maybe we add a privacy policy configuration that could be used by LibreTube/Piped? (it would be set up by instance owners)

**What does this even mean ? Elaborate please?**

**Comment by Bnyro** [comment]
Currently, all videos and images you can watch/see inside LibreTube are proxied through Piped (= loaded from Piped's servers, which itself loads them from YT), so you're not directly connected to YouTube.
When enabling the option to not proxy through Piped, the app will directly connect to YouTube's servers for loading images and video streams. So that should only be used if using a VPN. Your feed, search results, ... will still be provided by Piped though, it applies only for the thumbnail images and video stream sources (audio & video, e.g. opus or mpeg streams).
I hope that explains it, I'm not sure if and how it should be mentioned in the privacy policy.

**Comment by Bnyro** [comment]
> So YouTube will know your IP and what videos you are watching, I think that is important and should be reflected in the policy
>
Well, only if you don't use a VPN. It's explicitely mentioned that you should **NOT** use the option without VPN.

**Comment by heartlog** [comment]
> > So YouTube will know your IP and what videos you are watching, I think that is important and should be reflected in the policy
> 
> Well, only if you don't use a VPN. It's explicitely mentioned that you should **NOT** use the option without VPN.

Aggred

**Comment by heartlog** [comment]
> I think there still should probably be a note here about it

Well the feature is yet to be implemented. So, after this feature is added, then they will add information and maybe also warning.

**Comment by heartlog** [comment]
> TODO: Rephrase
> 
> > ### Instances & Data handling
> > if the user is not logged in to any Piped instances. In such cases, user data remains private and is not shared with any third parties.
> 
> Improve URLs:
> 
> ```
> <your.markdown.friendly.url>
> ```

?? Whats this supposed to mean ?

**Comment by heartlog** [comment]
> 1:
> 
> ```
> Instances & Data handling
> In case the user is not logged in to a Piped account on any instance, user data remains private and is not shared with any third parties
> ```

Whats wrong about it ?

> 
> 2: You write URLs like links: `[https://github.com](https://github.com)` But it's better to just use `<https://github.com>` in Markdown
Well ok

**Comment by heartlog** [comment]
That's what we call assertion and reason it's like if .... Then ....

**Comment by heartlog** [comment]
> This is a legal document, not a code or an algorithm, the text should be coherent. At least put a colon in there then

When a user is not logged in to any Piped instances, their data is kept private and is not shared with any third parties. This means that Piped does not collect, use, or disclose any personal information about the user when they are not logged in. Users can be assured that their privacy is protected by Piped's policies and practices, even when they are not actively using the platform. Piped values user privacy and takes measures to ensure that personal information is kept confidential and secure.

So called 'COHERENT' you were expecting ?

**Review by Bnyro — APPROVED** [review]

**Comment by Bnyro** [comment]
Looks good to me now, thanks for all your work!

**Pull request merged**